docs/concepts/workloads/pods/disruptions.md

```diff
@@ -0,0 +1,262 @@
+of time.  Examples are draining a node for maintenance or upgrade (learn how to
+[safely drain a node](/docs//tasks/administer-cluster/safely-drain-node.md)
+) and removing nodes from a cluster to scale it down (learn about
+[Cluster Autoscaling](/docs/tasks/administer-cluster/cluster-management/#cluster-autoscaler)
+).  In future releases of Kubernetes, a 
+[rescheduler](https://github.com/kubernetes/kubernetes/blob/master/docs/proposals/rescheduling.md)
+may also perform voluntary evictions.
+
+All sources of voluntary disruptions are optional with Kubernetes.
+Ask you cluster administrator or consult your cloud provider or distribution documentation
+to determine if any are configured for your cluster.  If none are configured, you can skip
+creating Pod Disruption Budgets.
+
+## Dealing with Disruptions
+
+Involuntary disruptions are typically infrequent,
+and so it is often sufficient to just accept them. 
+
+If higher availability is needed,
+then the application can be replicated.   (Learn about running replicated
+[stateless](/docs/tasks/run-application/run-stateless-application-deployment.md)
+and [stateful](/docs/tasks/run-application/run-replicated-stateful-application.md) applications.)
+Simultaneous failures of multiple nodes is less likely.  For even higher availability, use
+multi-zone clusters, spread applications across racks (using
+[anti-affinity](/docs/user-guide/node-selection/#inter-pod-affinity-and-anti-affinity-beta-feature))
+or across zones (if using a
+[multi-zone cluster](/docs/admin/multiple-zones).)
+
+The frequency of voluntary disruptions varies.  On a basic Kubernetes cluster, there are
+no voluntary disruptions at all.  However, your cluster admnistrator or hosting provider
+may run some additional services which cause voluntary disruptions.  For example,
+rolling out node software updates can cause voluntary updates.  Also, some implementations
+of cluster (node) autoscaling may cause voluntary disruptions to defragment and compact nodes.
+You cluster adminstrator or hosting provider should have documented what level of voluntary
+disruptions, if any, to expect.
+
+
+Kubernetes offers features to help run highly available applications at the same
+time as frequent voluntary disruptions.  We call this set of features
+*Disruption Budgets*.
+
+
+## How Disruption Budgets Work
+
+An Application Owner can create a `PodDisruptionBudget` object (PDB) for each application.
+A PDB limits the number pods of a replicated application that are down simultaneously due
+to voluntary disruptions.  For example, a quorum-based application would
+like to ensure that the number of replicas running is never brought below the
+number needed for a quorum, even temporarily. A web front end might want to
+ensure that the number of replicas serving load never falls below a certain
+percentage of the total, even briefly.  
+
+Cluster management tools can use the `Eviction API` to "safely delete" pods
```

Comment:
Hyperlink to... something please.  We're using a Proper Name for the first time.  Maybe just link to safely-drain-node.md?